[jsoncons] Compilation error

[ 52%] Performing install step for 'jsoncons'
Install the project...
-- Install configuration: "Release"
-- Installing: /usr/local/include/jsoncons
CMake Error at cmake_install.cmake:46 (file):
  file INSTALL cannot make directory "/usr/local/include/jsoncons":
  Permission denied.


Solution: Jsoncons needs a proper external project .cmake file.

--- a/superbuild/projects_modules/jsoncons.cmake
+++ b/superbuild/projects_modules/jsoncons.cmake
@@ -40,7 +40,8 @@ ExternalProject_Add(${ep}
   DEPENDS ${${ep}_dependencies}
   CMAKE_GENERATOR ${gen}
   CMAKE_GENERATOR_PLATFORM ${CMAKE_GENERATOR_PLATFORM}
-  CMAKE_ARGS ${cmake_args} 
+  CMAKE_ARGS ${cmake_args}
+  INSTALL_COMMAND ""
   )
 ## #############################################################################
 ## Set variable to provide infos about the project

